### ISSUE TITLE
VAE_SCALING_FACTOR inconsistency of image during Cogvideo1.5 training and inference

### ISSUE THREAD

**Issue by CN-makers** [a-r-r-o-w/cogvideox-factory#110]
### System Info / 系統信息

Diffusers 0.32.0.dev

### Information / 问题信息

- [X] The official example scripts / 官方的示例脚本
- [X] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

In the https://github.com/a-r-r-o-w/cogvideox-factory/blob/main/training/cogvideox_image_to_video_lora.py. 743
`image_latents = image_latent_dist.sample() * VAE_SCALING_FACTOR`
i think for cogvideo 1.5 this should be 
`image_latents = image_latent_dist.sample() * 1 /VAE_SCALING_FACTOR`
because in the diffusers  0.32.0.dev
https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/cogvideo/pipeline_cogvideox_image2video.py
385 to 390

```
    if not self.vae.config.invert_scale_latents:
        image_latents = self.vae_scaling_factor_image * image_latents
    else:
        # This is awkward but required because the CogVideoX team forgot to multiply the
        # scaling factor during training :)
        image_latents = 1 / self.vae_scaling_factor_image * image_latents
```

### Expected behavior / 期待表现

I hope this part of the code can be modified and seen by more people.

**Comment by sayakpaul** [comment]
@a-r-r-o-w @zRzRzRzRzRzRzR the proposed fix appears correct to me.

**Comment by JonathanLi19** [comment]
Same question here.

**Comment by sayakpaul** [comment]
We have fixed it in the new trainer API. Please take a look. https://github.com/a-r-r-o-w/finetrainers

**Comment by JonathanLi19** [comment]
> We have fixed it in the new trainer API. Please take a look. https://github.com/a-r-r-o-w/finetrainers

Thanks! Just to make sure (since for now only T2V Lora training is supported): if I want to train a CogVideoX1.5-5B-I2V model, **I should not multiply image_latents by self.vae_scaling_factor_image or (1 / self.vae_scaling_factor_image)**, right?
Just kee the original image_latents.

**Comment by sayakpaul** [comment]
I think it should be configured based on 
https://github.com/a-r-r-o-w/finetrainers/blob/7b569daf55e7c633e085a740833f44f2863a62fa/finetrainers/cogvideox/cogvideox_lora.py#L139C5-L150C36

**Comment by JonathanLi19** [comment]
Yes, but it's weird since CogVideoX1.5-5B-I2V will do` image_latents = 1 / self.vae_scaling_factor_image * image_latents`during inference.
But, in https://github.com/a-r-r-o-w/finetrainers/blob/7b569daf55e7c633e085a740833f44f2863a62fa/finetrainers/cogvideox/cogvideox_lora.py#L139C5-L150C36 it seems that we don't multiply 1 / self.vae_scaling_factor_image

> I think it should be configured based on https://github.com/a-r-r-o-w/finetrainers/blob/7b569daf55e7c633e085a740833f44f2863a62fa/finetrainers/cogvideox/cogvideox_lora.py#L139C5-L150C36

**Comment by CN-makers** [comment]
Im still confused. If, as the official team mentioned, the self.vae_scaling_factor_image should indeed be applied directly, then the output should not be scaled. However, in the diffusers code, it is multiplied by 1/self.vae_scaling_factor_image, suggesting that the official team might have mistakenly used 1/self.vae_scaling_factor_image instead. Additionally, I believe this issue only pertains to image_latents and not to video_latents. This is because, in the inference code, there is no distinction between 1.5 and 1.0 for video_latents. Therefore, video_latents should still be multiplied by self.vae_scaling_factor_image, while image_latents should be multiplied by 1/self.vae_scaling_factor_image to maintain consistency between training and inference.

**Comment by CN-makers** [comment]
i think maybe it's diffusers wrong? the reference can be seen here. https://github.com/THUDM/CogVideo/issues/570#issuecomment-2550667997

**Comment by Yuancheng-Xu** [comment]
So to summarize, during both training and inferencing with CogVideoX-1.5 (both T2V and I2V), we do not multiply image_latents at all. Is that correct?

**Comment by hbb1** [comment]
> So to summarize, am I understanding it correctly? I am not very sure.
> 
> For CogvideoX-1.5-T2V: nothing needs to be changed.
> 
> For CogvideoX-1.5-I2V:
> 
> * In fine-tuning script, in order to be consistent with how CogVideoX team trained the model,  image_latents should not be scaled at all during training. Note that the video_latents should still be scaled. In particular, [here](https://github.com/a-r-r-o-w/finetrainers/blob/da9d7d92a5e71aec82b7a1a25d97631458212bd5/finetrainers/models/cogvideox/lora.py#L147) the function is for preparing latents for the videos: I think the video_latents should always be scaled (instead of the current implementation). It is the image_latents that we need to take care of.
> * In pipeline (inference) script, image_latents should not be scaled at all either.

I have the same question here. So in the fine-tuning script, should we un-scale the image_latent?
```python
# image_latents = image_latent_dist.sample() * VAE_SCALING_FACTOR
image_latents = image_latent_dist.sample()
```
